ProxyTests/LoggerTest.swift

```diff
@@ -1,27 +1,22 @@
-//
-//  LoggerTest.swift
-//  SplitTunnelProxyTests
-//
-//  Created by John Mair on 19/01/2024.
-//  Copyright © 2024 PIA. All rights reserved.
-//
-
 import Quick
 import Nimble
 
 @testable import SplitTunnelProxyExtensionFramework
 class LoggerTest: QuickSpec {
     override class func spec() {
         describe("LoggerTest") {
-            context("updateLogger()") {
+            // Be careful because Xcode runs the tests in alphabetical order,
```

Comment:
Is it possible to fix this test to make it order independent?

Reply:
yes, by moving it to a new file. that is probably the best solution, the fact that test order matters in a single test is not ideal

Reply:
I did exactly that